.gitignore

```diff
@@ -0,0 +1 @@
+.env
```

Comment:
Where do you have your .env file stored in? It might make more sense to have the `.env` file within the `oldScripts` folder because we might have multiple `.env` files if we add more folders later in development, similar to GradeView. The gitignore could go here, or moved to `oldScripts`, but I suggest the `.env` should go in the `oldScripts` folder

Reply:
Absolutely right! Will change this to oldScripts/.env

Reply:
Ping me on slack once you've resolved the comments

Reply:
Thanks for your review, Mehul! All comments resolved

Reply:
Just realized I didn't press the resolved buttons; will do that now.